app/app/static/custom/ourscript.js

```diff
@@ -352,10 +352,45 @@ window.onload = async (event) => {
 
 // Submit button for export page
 $('#dateSubmit').on('click', function(e) {
+	// First day of operation
+	const firstDay = "2021-09-28";
 	// Get values from page
-	const startDate = document.getElementById("startDate").value;
-	const endDate = document.getElementById("endDate").value;
+	const dateRange = $("input[name='dateRangeSelector']:checked").val()
+	let startDate;
+	// Get current date
+	let today = new Date();
+	let start = new Date(today);
+	let endDate = today.getFullYear()+'-'+(today.getMonth()+1)+'-'+today.getDate();
+	// Generate correct dates based of input
+	switch(dateRange) {
+		case 'Last Day':
+			start.setDate(start.getDate()-1)
+			startDate = start.getFullYear()+'-'+('0' + (start.getMonth()+1)).slice(-2)+'-'+start.getDate();
+			break;
+		case 'Last Week':
+			start.setDate(start.getDate()-7)
+			startDate = start.getFullYear()+'-'+('0' + (start.getMonth()+1)).slice(-2)+'-'+start.getDate();
+			break;
+		case 'Last Month':
+			start.setMonth(start.getMonth()-1)
+			startDate = start.getFullYear()+'-'+('0' + (start.getMonth()+1)).slice(-2)+'-'+start.getDate();
+			break;
+		case 'Last Year':
+			start.setFullYear(start.getFullYear()-1)
+			startDate = start.getFullYear()+'-'+('0' + (start.getMonth()+1)).slice(-2)+'-'+start.getDate();
+			break;
+		case 'All Time':
+			startDate = firstDay;
+			break;
```

Comment:
consider using Javascript string template for this

Reply:
or maybe even using the iso string and splice it? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString

I am sure there must be a standard built-in way to get the datestring

Reply:
wasn't aware of the toISOstring method. That makes it much cleaner